File: backend/src/main/java/gov/inl/LIVE/common/JsonError.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package gov.inl.LIVE.common;

/**
 *
 * @author monejh
 */
public class JsonError extends ITypeName
{
    private String error;

    public JsonError()
    {
    }

    
    public JsonError(String error)
    {
        this.error = error;
    }

    
    
    public String getError()
    {
        return error;
    }

    public void setError(String error)
    {
        this.error = error;
    }
    
    
}

File: backend/src/main/webapp/resources/angular2/menu/app/jsonerror.ts
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
 
 import { ITypeName } from './ITypeName';

export class JsonError implements ITypeName
{
    typeName: string;
    error: string;
    
    constructor(error: string)
    {
        this.error = error;
    }
    
}

